Source/Core/Common/LdrWatcher.cpp

```diff
@@ -116,8 +116,8 @@ void LdrDllNotifier::Install(LdrObserver* observer)
   if (!initialized)
     return;
   void* cookie{};
-  if (!NT_SUCCESS(LdrRegisterDllNotification(0, DllNotificationCallback,
-                                             static_cast<PVOID>(observer), &cookie)))
+  if (LdrRegisterDllNotification(0, DllNotificationCallback, static_cast<PVOID>(observer), &cookie)
+      < 0)
```

Comment:
Just like the other one, I feel like this is a step backwards (even though this one is arguably shorter.) But the point isn't necessarily saving characters, it is primarily readability for when someone revisits this piece of code at a later stage.

A `![NT_]Success(Call(...))` spells it out, a `Call(...) < 0` leaves me guessing why this isn't `Call(...) != 0` or `Call(...) <= 0`.